research/src/pages/tabs.research.mdx

```diff
@@ -0,0 +1,19 @@
+---
+name: Tabs (Working Draft)
+path: /components/tabs.research
+pathToProposal: /components/tabs
+---
+
+import Anatomy from '../components/anatomy'
+import Concepts from '../components/concepts'
+import ComponentCoverage from '../components/component-coverage'
+
+<ComponentCoverage component="Tabs" />
+
+## Anatomy
```

Comment:
I should just remove this from the template as this ends up in the proposal and is an artifact that is left over from early on